scripts/devnet-init.sh

```diff
@@ -50,7 +55,8 @@ ethContractExists() {
 }
 
 wait "curl --output /dev/null --silent --fail $alphNodeUrl/addresses/$alphNativeTokenBridge/group"
-wait "ethContractExists $ethNativeTokenBridge"
+wait "evmContractExists $ethNativeTokenBridge $ethNodeUrl"
```

Comment:
[Nit] `ethContractExists` & `bscContractExists`

Reply:
The `evmContractExists` is a function name, it can be used for both ETH and BSC.

Reply:
Sorry, i misread, thought this is the printout.